src/test/groovy/no/fint/portal/nam/AuthorizationPolicyServiceSpec.groovy

```diff
@@ -0,0 +1,45 @@
+package no.fint.portal.nam
+
+
+import spock.lang.Specification
+
+class AuthorizationPolicyServiceSpec extends Specification {
+
+    private AuthorizationPolicyService authorizationPolicyService;
+
+    void setup() {
+        authorizationPolicyService = new AuthorizationPolicyService(
+                adapterScope: "fint-client",
+                clientScope: "fint-client",
+                clientAttribute: "fintClientComponents",
+                adapterAttribute: "fintAdapterComponents"
+        )
+    }
+
+    def "Creating a client policy should return a client policy"() {
+        when:
+        def clientPolicy = authorizationPolicyService.createClientPolicy("test", "ou=utdanning_elev,ou=components,o=fint")
+
+        then:
+        clientPolicy.policyName == "a_client_test"
+        clientPolicy.rule.size() == 3
+        clientPolicy.rule[0].actionList.action.size() == 1
+        clientPolicy.rule[0].conditionList.conditionSet.size() == 1
+
+        //def jsonDefaultOutput = new JsonGenerator.Options().excludeNulls().build()
+        //def json = jsonDefaultOutput.toJson(clientPolicy)
+        //println json
```

Comment:
This is not legal in `then:`, but could be added to a second `when:` with `then: noExceptionThrown()`